src/components/ipv-callback/tests/ipv-callback-integration.test.ts

```diff
@@ -46,7 +46,7 @@ describe("Integration:: ipv callback", () => {
 
     const requestPath = PATH_NAMES.IPV_CALLBACK + "?code=" + "12345";
 
-    await request(app, (test) => test.get(requestPath).expect(200), {
+    await request(app, (test) => test.get(requestPath).expect(302), {
```

Comment:
I think it'd be lightly more expressive here to also say where it's redirected to as well as status. Also might want to update test description here too

Reply:
Good shout, updated